sys-fs/cryptsetup/cryptsetup-2.7.5.ebuild

```diff
@@ -20,12 +20,14 @@ fi
 CRYPTO_BACKENDS="gcrypt kernel nettle +openssl"
 # we don't support nss since it doesn't allow cryptsetup to be built statically
 # and it's missing ripemd160 support so it can't provide full backward compatibility
-IUSE="${CRYPTO_BACKENDS} +argon2 fips nls pwquality ssh static static-libs test +udev urandom"
+IUSE="${CRYPTO_BACKENDS} +argon2 fips nls pwquality passwdqc ssh static static-libs test +udev urandom"
```

Comment:
Let's do the latest version only and do an ~arch revbump for it, just given it's a critical package.